src/Microsoft.ML.Ensemble/OutputCombiners/BaseStacking.cs

```diff
@@ -15,7 +15,7 @@
 
 namespace Microsoft.ML.Ensemble.OutputCombiners
 {
-    internal abstract class BaseStacking<TOutput> : IStackingTrainer<TOutput>
+    internal abstract class BaseStacking<TOutput> : IStackingTrainer<TOutput>, ICanSaveModel
```

Comment:
>, ICanSaveModel [](start = 77, length = 15)

Geez, every time I look at this assembly it's something else... thanks for fixing this @artidoro!